ONSdigital/rm-sample-service
# upgrading springboot and adding custom liquibase

# What and why?
Spring boot needed to be upgraded to version 2.6.7 like the other Java services.
It turned out that there were a few liquibase changes that weren't in the changelog-master that were being run that were throwing errors.  Preprod and prod had no record of them being run so it should be safe to delete them.

# How to test?

- Deploy fresh copy of your environment
- Delete the `sample` schema using something like `DROP SCHEMA IF EXISTS sample CASCADE;`
- Deploy this PR to your environment. It should recreate the sample schema
- Run acceptance tests, they should pass
- Go on r-ops and upload a sample file to make sure it works
- `kc rollout restart deployment sample` to redeploy service, it should boot up correctly and the data should still remain

# Trello
[Card](https://trello.com/c/Oiple9r6)
